ibm/mas_devops/roles/dro/tasks/gencfg/migrate.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: "gencfg-migrate : debug UDS BASCfgs"
+  debug:
+    msg:
+      - "name:        {{item.metadata.name}}"
```

Comment:
Can we standardize the debug following other samples as in https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d/tasks/install-cp4d.yml#L46

Reply:
@andrercm done. i have updated the debug msg

Reply:
@andrercm done. i have updated the debug msg